.github/test_e2e_op_args_chain_1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "args": {
+    "zkevm_rollup_chain_id": 20201
+  },
+  "optimism_package": {
+    "predeployed_contracts": true,
+    "chains": {
+      "001": {
+        "proposer_params": {
+          "enabled": false
+        },
+        "network_params": {
+          "network_id": "20201",
+          "seconds_per_slot": 1
+        }
+      }
+    },
+    "observability": {
+      "enabled": false
```

Comment:
I think you can remove this part as the OP parser has been fixed and observability is disabled by default: https://github.com/0xPolygon/kurtosis-cdk/blob/2c749117bb7078c361cf31542968224e00791ea3/src/package_io/op_input_parser.star#L86

Reply:
Fixed https://github.com/agglayer/aggkit/pull/1128/commits/f4e3fce4d0cdb8df5009a658592198c191f26596